<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<!-- robot_pose_ekf -->
	<node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf">
  		<param name="output_frame" value="$(arg output_frame)"/>
		<param name="base_footprint_frame" value="base_footprint"/>
		<param name="freq" value="30.0"/>
		<param name="sensor_timeout" value="1.0"/>  
		<param name="odom_used" value="true"/>
		<param name="imu_used" value="false"/>
		<param name="vo_used" value="false"/>

  		<remap from="imu_data" to="imu" />
  		<!-- default subscribe /imu_data  /odom  /vo-->
	</node>
</launch>
